modules/exploits/firewall/extrabacon.rb

```diff
@@ -203,7 +221,6 @@ def hex(buf)
 
   def setup
     sc_file = File.join(Msf::Config.data_directory, "exploits", "EXTRABACON", "versions", "shellcode_" + target['Version'] + ".rb")
-    targ_file = File.join(Msf::Config.data_directory, "exploits", "EXTRABACON", "exba_targets.rb")
 
     if datastore['Verbose'] == true
       print_status("Using config file: #{sc_file}")
```

Comment:
The normal way to handle verbose print is with: ``vprint_status`` (should be in the stock mixins here)